config/crd/bases/starlingx.windriver.com_platformnetworks.yaml

```diff
@@ -150,10 +94,9 @@ spec:
                 - other
                 type: string
             required:
-            - allocation
-            - prefix
-            - subnet
-            - type
+            - associatedAddressPools
+            - ipAllocationType
```

Comment:
to align with the concept in the sysinv, can we make it to "dynamic", and use boolean value?